packages/networks/src/networks/fuse.ts

```diff
@@ -0,0 +1,74 @@
+import { NetworkConfig } from '@unlock-protocol/types'
+
+export const fuse: NetworkConfig = {
+  blockScan: {
+    url: (address: string) => `https://blockscan.com/address/${address}`,
+  },
+  chain: 'fuse',
+  description: 'A public testnet for Fuse.',
+  explorer: {
+    name: 'Fuse Explorer',
+    urls: {
+      address: (address: string) =>
+        `https://explorer.fuse.io/address/${address}`,
+      base: `https://explorer.fuse.io/`,
+      token: (address: string, holder: string) =>
+        `https://explorer.fuse.io/token/${address}?a=${holder}`,
+      transaction: (hash: string) => `https://explorer.fuse.io/tx/${hash}`,
+    },
+  },
+  featured: false,
+  fullySubsidizedGas: true,
+  hooks: {},
+  id: 122,
+  isTestNetwork: false,
+  keyManagerAddress: '',
+  maxFreeClaimCost: 1000,
+  multisig: '',
+  name: 'Fuse Mainnet',
+  nativeCurrency: {
+    coingecko: 'fuse',
+    decimals: 18,
+    name: 'FUSE',
+    symbol: 'FUSE',
+    wrapped: '0x0BE9e53fd7EDaC9F859882AfdDa116645287C629',
+  },
+
+  previousDeploys: [],
+
+  provider: 'https://rpc.fuse.io/',
+
+  publicLockVersionToDeploy: 14,
+
+  publicProvider: 'https://rpc.fuse.io/',
+
+  startBlock: 31355828,
+  subgraph: {
+    endpoint:
+      'https://api.studio.thegraph.com/query/87693/unlock-protocol-fuse/version/lastest', // this is given to you by the graph after deploying
```

Comment:
I don't think this has been deployed?

Reply:
![image](https://github.com/user-attachments/assets/11f76ac3-fed1-4453-b10d-0ed50ad8defb)